woocommerce/woocommerce
# Add check for tasklist layout experiment

### All Submissions:






### Changes proposed in this Pull Request:

Adds experiment treatment checks for the new task list sections.
<img width="519" alt="Screen Shot 2022-04-13 at 1 42 24 PM" src="https://user-images.githubusercontent.com/10561050/163242878-bfb27975-38b3-4fdd-887b-a417c292000b.png">
<img width="559" alt="Screen Shot 2022-04-13 at 1 41 56 PM" src="https://user-images.githubusercontent.com/10561050/163242884-ab0cbd76-83fc-4c29-a735-e040ca20c8b4.png">
<img width="1004" alt="Screen Shot 2022-04-13 at 1 41 41 PM" src="https://user-images.githubusercontent.com/10561050/163242891-a27e46b7-0760-41dd-af3f-a382d17a5cb9.png">



Closes #32150  .

### How to test the changes in this Pull Request:

1. Install and enable the [WCA test helper](https://github.com/woocommerce/woocommerce-admin-test-helper/)
2. Go to the homescreen and note that the old task list is shown
3. Navigate to Tools->WCA Test Helper->Experiments
4. Toggle the `woocommerce_tasklist_setup_experiment_1_2022_04` experiment to `treatment`.  (note the month matches this month which will make this experiment work with `_05` once may hits)
5. Navigate to the homescreen
6. Check that the experiment 1 task list is shown
7. Make sure the task list reminder bar is shown (delete the `woocommerce_task_list_reminder_bar_hidden` option if necessary using the helper plugin)
8. Toggle `woocommerce_tasklist_setup_experiment_1_2022_04` back to `control`.
9. Toggle `woocommerce_tasklist_setup_experiment_2_2022_04` to `treatment`
10. Navigate to the homescreen
11. Check the setup experiment 2 is shown
12. Check that the task reminder bar is no longer shown

### Other information:




### Changelog entry

Changelog not needed as [this work](https://github.com/woocommerce/woocommerce/pull/32302) is being included in this cycle/version.

### FOR PR REVIEWER ONLY:
